plugin/pkg/auth/authorizer/node/graph_populator.go

```diff
@@ -212,7 +212,7 @@ func (g *graphPopulator) deleteVolumeAttachment(obj interface{}) {
 	if tombstone, ok := obj.(cache.DeletedFinalStateUnknown); ok {
 		obj = tombstone.Obj
 	}
-	attachment, ok := obj.(*api.PersistentVolume)
+	attachment, ok := obj.(*corev1.PersistentVolume)
```

Comment:
this is a pre-existing copy/paste bug... this should be `storagev1beta1.VolumeAttachment`

Reply:
yeah, let me deal with that one separately. it will need picking back to older releases

Reply:
will rebase onto that